src/test/groovy/org/araneforseti/boundary/TestUtil.groovy

```diff
@@ -3,8 +3,13 @@ package org.araneforseti.boundary
 import org.araneforseti.boundary.definitions.Definition
 import org.araneforseti.boundary.definitions.ExpectedResponse
 import org.araneforseti.boundary.fields.Field
+import org.araneforseti.boundary.util.MessageConfiguration
 
 class TestUtil {
+
+    public static final String messageName = "this is the field name"
+    public static final String fieldType = "this is the field type"
```

Comment:
These belong in the tests which care about them because a test should be immediately obvious what it is setting up and what the consequences of the setup are